docs/reference/garbage-collection.md

```diff
@@ -148,7 +156,29 @@ spark-submit --class io.treeverse.clients.GarbageCollector \
   -c spark.hadoop.fs.s3a.secret.key=<S3_SECRET_KEY> \
   <APPLICATION-JAR-PATH> \
   example-repo us-east-1
-```
+  ```
+  </div>
+
+  <div markdown="1" id="azure-option">
+The relevant Spark client for running GC on Azure is `lakefs-spark-client-312-hadoop3` only. This client is compiled for Spark 3.1.2 with Hadoop 3.2.1, but can work for other Spark versions and higher Hadoop versions.
+
+You should run the following command to make the garbage collector start running:
+
+  ```bash
+spark-submit --class io.treeverse.clients.GarbageCollector \
+  -c spark.hadoop.lakefs.api.url=https://lakefs.example.com:8000/api/v1  \
+  -c spark.hadoop.lakefs.api.access_key=<LAKEFS_ACCESS_KEY> \
+  -c spark.hadoop.lakefs.api.secret_key=<LAKEFS_SECRET_KEY> \
+  -c spark.hadoop.fs.azure.account.key.<AZURE_STORAGE_ACCOUNT>.dfs.core.windows.net=<AZURE_STORAGE_ACCESS_KEY> \
+  s3://treeverse-clients-us-east/lakefs-spark-client-312-hadoop3/0.1.8/lakefs-spark-client-312-hadoop3-assembly-0.1.8.jar \
+  example-repo
+  ```
+In case you don't have `hadoop-azure` package as park of your environment, you should add the package to your spark-submit with `--packages org.apache.hadoop:hadoop-azure:3.2.1`
```

Comment:
```suggestion
In case you don't have `hadoop-azure` package as part of your environment, you should add the package to your spark-submit with `--packages org.apache.hadoop:hadoop-azure:3.2.1`
```